view/frontend/templates/flowjs.phtml

```diff
@@ -13,6 +13,9 @@
 ?>
 <?php if ($block->isFlowEnabled()) : ?>
     <script type="text/javascript">
+    define('flow_organization_id', [], () => {
```

Comment:
@flowio-david I'm not familiar with a normal Magento eco-system, but I'm presuming `define` is available on the page where this template is written to? It is in the case of Primera.